### PR TITLE
[WIP]Proxy doc for 3.2

### PR DESCRIPTION
**Pull request by rosieyohannan**
https://circleci.atlassian.net/browse/SERVER-1400

### CODE REVIEW

File: jekyll/_cci2/server-3-operator-proxy.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+== Installation and configuration
+There are two stages to installing CircleCI server behind a proxy. First at the point of installation the proxy addresses need to be specified, along with any addresses that should not be behind the proxy.
+
+=== Installing behind a proxy
+The installation process is described in detail in the https://circleci.com/docs/2.0/server-3-install/[CircleCI Server v3.x Installation guide]. Both proxy and non-proxy addresses should be supplied using the arguments described https://kots.io/kotsadm/installing/online-install/#proxies[here]. The installation command should be in the form:
+
+[source,bash]
+----
+kubectl kots install circleci-server --http-proxy <my-http-proxy-uri> --https-proxy <my-https-proxy> --no-proxy <my-no-proxy-list>
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-proxy.adoc:25]: Can we take out the HTTPS argument here due to the known issue? @nsmith5

File: jekyll/_cci2/server-3-operator-proxy.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+[INSERT SCREENSHOT]
+
+
+== Known limitations
+
+* The JVM only accepts proxies that run over HTTP not HTTPS and therefore proxy URIs must be of the form `http://user:password@host:port` rather than `https://user:password@host:port`.
+* If your GitHub instance is running outside of the proxied environment (either GitHub.com or GitHub enterprise), you must ensure that SSH traffic from our application (inside the Kubernetes cluster) and from our Nomad node can reach your instance without additional configuration. Our SSH agents do not respect proxy settings because it uses a different network protocol. If a proxy is the only way that any traffic can reach outside the proxied environment this means it will block SSH traffic and our application will fail.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-proxy.adoc:37]: Do we need to provide any details on how they will ensure this or is that up to the customer? @nsmith5

File: jekyll/_cci2/server-3-operator-proxy.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+* The load balancer endpoints must be added to the no-proxy list for the following services: `output processor` and `vm-service`. This is because the no-proxy list is shared between the application and build-agent. The application and build-agent are assumed to be behind the same firewall and therefore cannot have a proxy between them.
+* The KOTS admin console cannot be upgraded if proxy setting were set. The proxy settings will be deleted and cause the KOTS admin console to break.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-proxy.adoc:39]: I guess we need something in any upgrade guide we produce to account for this for customers that install behind a proxy?

File: jekyll/_cci2/server-3-operator-proxy.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+* Often, a custom image is needed for VM service that has proxy configured! (Check with Makoto Mizukami   about this and how customers have set this up before)
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-operator-proxy.adoc:40]: @makotom can you help me out with this one?

File: jekyll/_cci2/server-3-operator-proxy.adoc (hunk 1 above)
Comment (nsmith5) [reply jekyll/_cci2/server-3-operator-proxy.adoc:25]: Nope. See https://circleci.atlassian.net/browse/SERVER-1400?focusedCommentId=158461

File: jekyll/_cci2/server-3-operator-proxy.adoc (hunk 2 above)
Comment (nsmith5) [reply jekyll/_cci2/server-3-operator-proxy.adoc:37]: Up to the customer I'd say. There are many ways to make this possible and its coupled to the way they organized their network. Its hard to give generic advise on how to achieve this

File: jekyll/_cci2/server-3-operator-proxy.adoc (hunk 3 above)
Comment (nsmith5) [reply jekyll/_cci2/server-3-operator-proxy.adoc:39]: Perhaps something to add to the change log for this release. Once the problem is fixed up stream with Kots we can add another note to that change log and remove this documentation

File: jekyll/_cci2/server-3-operator-proxy.adoc (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server-3-operator-proxy.adoc:25]: thank you!

File: jekyll/_cci2/server-3-operator-proxy.adoc (hunk 3 above)
Comment (rosieyohannan) [reply jekyll/_cci2/server-3-operator-proxy.adoc:39]: Yep I'll add to the changelog as a known issue, thanks!

**Pull request merged**